Application/Business/AttributeGroup/AttributeGroupServices.cs

```diff
@@ -22,34 +27,32 @@ public async Task<int> Count()
     {
         return await _attributeRepo.CountAsync();
     }
-    public async Task Create(AttributeGroupEntity attributeGroup)
+    public async Task Create(AttributeGroupEntity attributeGroup){
+        await _attributeRepo.CreateAsync(attributeGroup);
+    }
+    public async Task Update(Guid attributeGroupid, AttributeGroupEntity attributeGroup)
     {
-        var attList = List();
-        var check = false;
-        foreach (var attgrp in attList.Result)
+        var target = await GetById(attributeGroupid);
+        if(target is not null)
         {
-            if (attgrp.Name == attributeGroup.Name)
-            {
-                check = true;
-                break;
-            }
+            await _attributeRepo.UpdateAsync(target);
         }
-        if (!check)
+        else
```

Comment:
if (target is null) throw err 
do logic (remove else condition)

Reply:
Remove else
```
public async Task Update(Guid attributeGroupid, AttributeGroupEntity attributeGroup)
    {
        var target = await GetById(attributeGroupid);
        if(target is null)
        {
            throw new NotFoundException("Attribute group not exist");
        }
        await _attributeRepo.UpdateAsync(target);
    }
```